README.md

```diff
@@ -154,13 +163,14 @@ We do believe that the knowledge here,
 difference for **both employees and hiring managers**.
 
 One of the difficulties for candidates negotiating equity compensation is that they may
-have less information about what they are worth than the person hiring them.
-Companies talk to many candidates and often pay for or have access to expensive
+have less information about [what they are worth](#equal-treatment) than the person hiring
```

Comment:
This too seems like we could do better in terms of relevance of the link. I think the equal treatment section is all about equal treatment, not worth.

Reply:
Fixed.